client-service/src/main/java/com/registry/clientservice/dto/ClientDTO.java

```diff
@@ -0,0 +1,13 @@
+package com.registry.clientservice.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class ClientDTO {
```

Comment:
The id of the client should probably be included here as well?

Reply:
Nope, i don't use the id in any controller/service method.